MAUI/ListView/item-drag-and-drop.md

```diff
@@ -0,0 +1,485 @@
+---
+layout: post
+title: Item Reordering in .NET MAUI ListView control | Syncfusion
+description: Learn here all about Item Reordering support in Syncfusion .NET MAUI ListView (SfListView) control and more.
+platform: MAUI
+control: SfListView
+documentation: ug
+---
+
+# Item Reordering in .NET MAUI ListView (SfListView)
+
+The `SfListView` allows reordering by dragging and dropping items. It supports displaying the customized view in a template while dragging the item. It can be enabled by setting the `SfListView.DragStartMode` property to `OnHold`. The drag and drop options are listed as follows:
```

Comment:
The `SfListView` allows reordering by dragging and dropping items. It supports displaying the customized view in a template while dragging the item. It can be enabled by setting the `SfListView.DragStartMode` property to `OnHold`. The drag-and-drop options are listed as follows:

Reply:
Addressed the concerns provided.